var/spack/repos/builtin/packages/tcsh/package.py

```diff
@@ -0,0 +1,62 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
```

Comment:
I have also updated this. Should I also submit a PR to update the template that `spack create` uses?